suite/env/.env.dev

```diff
@@ -0,0 +1,3 @@
+ANCHOR_INTERVAL_MIN=780
+TEST_INTERVAL_MIN=360
```

Comment:
I think we can get rid of `TEST_INTERVAL_MIN` from all the env files. This was used to control the frequency of the tests but doesn't apply anymore in this new approach.